smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy

```diff
@@ -27,7 +27,7 @@ definition(
 )
 
 preferences() {
-	section("Choose thermostat... ") {
+	section("Choose thermostat...") {
```

Comment:
are there translations for these lines coming later?

Reply:
The translations are there but the space was messing the key

Reply:
this key is in the translations files twice